frontend/src/components/home/overview/overview.module.css

```diff
@@ -27,7 +27,7 @@
 }
 
 .dash {
-  width: 90px;
+  width: 450px;
```

Comment:
let's keep `width:300px`

Reply:
Updated:
![image](https://user-images.githubusercontent.com/67599130/107478002-b7afc580-6bb3-11eb-822d-c6b4c9455670.png)
![image](https://user-images.githubusercontent.com/67599130/107478023-c0080080-6bb3-11eb-9470-a4d57c0c4cb8.png)

Reply:
@Kajol-Kumari I'm done with the edit! Please review thanks! 😄